code/controllers/subsystems/atoms.dm

```diff
@@ -8,6 +8,9 @@ SUBSYSTEM_DEF(atoms)
 	init_order = SS_INIT_ATOMS
 	flags = SS_NO_FIRE | SS_NEEDS_SHUTDOWN
 
+	var/override_populate_parts = FALSE
```

Comment:
Possibly a brief comment explaining that it is for downstream use rather than Neb use?